### ISSUE TITLE
Update Send-SMS-with-Twilio-from-IBM-i example

### ISSUE THREAD

**Issue by abmusse** [IBM/ibmi-oss-examples#52]
Looks like the [Send-SMS-with-Twilio-from-IBM-i] example)(https://github.com/IBM/ibmi-oss-examples/tree/master/nodejs/Send-SMS-with-Twilio-from-IBM-i) uses older versions of `itoolkit` and `twilio` but the example does not include a package.json file. We should update the example to use the latest version of these packages.

@ThePrez IIRC you were also working on twilio example similar to this one. Do you wanna add your changes to this one?

**Comment by ThePrez** [comment]
Yes, I wrote a Twilio example derived from this one which integrates with RPG in a variety of ways. The code is at https://github.com/ThePrez/OSS-integration-pptx 

It needs lots of documentation work, as you can see! When it's more complete, this repo is the proper place for it. Assigning to me.
